newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -166,6 +166,8 @@ export default class PixiResourcesLoader {
     const loadedTexture = loadedTextures[resourceName];
     if (loadedTexture && loadedTexture.textureCacheIds) {
       await PIXI.Assets.unload(loadedTexture.textureCacheIds);
+      // Needed for video objects. Optional for other object types.
```

Comment:
This comment is confusing me. How come you know this is a video object? `loadedTextures` is only for videos? or is it because you have `textureCacheIds` ?

Reply:
Thanks for the challenge. I cleaned that and added a comment